packages/web/src/components/gcds-link/stories/gcds-link.stories.js

```diff
@@ -0,0 +1,211 @@
+      },
+    },
+  },
+};
+
+const Template = args =>
+  `
+This is an example of
+<!-- Web component code (Angular, Vue) -->
+<gcds-link ${args.display ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target ? `target="${args.target}"` : null
+  } ${args.size != 'regular' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</gcds-link>
+
+<!-- React code -->
+<GcdsLink ${args.display ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target ? `target="${args.target}"` : null
+  } ${args.size != 'regular' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</GcdsLink>
+link.
+`.replace(/ null/g, '');
+
+const TemplateSizes = () => `
+<!-- Web component code (Angular, Vue) -->
+<gcds-link href="" size="regular">This is a regular link.</gcds-link>
+<gcds-link href="" size="small">This is a small link.</gcds-link>
+
+<!-- React code -->
+<GcdsLink href="" size="regular">This is a regular link</GcdsLink>
+<GcdsLink href="" size="small">This is a small link</GcdsLink>
+`;
+
+const TemplatePlayground = args => `
+<gcds-link ${args.display ? `display="${args.display}"` : null} ${
+  args.href ? `href="${args.href}"` : null
+} ${args.rel ? `rel="${args.rel}"` : null} ${
+  args.target ? `target="${args.target}"` : null
+} ${args.size != 'regular' && args.size ? `size="${args.size}"` : null} ${
+  args.external ? `external` : null
+} ${args.download ? `download="${args.download}"` : null} ${
+  args.type ? `type="${args.type}"` : null
+}>  ${args.default}
+</gcds-link>
+`;
+
+// ------ Link Default ------
+export const Default = Template.bind({});
+Default.args = {
+  display: '',
+  href: '#',
+  rel: '',
+  target: '',
+  size: 'regular',
+  external: false,
+  download: '',
+  type: '',
+  default: 'a GCDS Link component',
+};
+
+// ------ Link events & props ------
+export const Props = Template.bind({});
+Props.args = {
+  display: '',
+  href: '#',
+  rel: '',
+  target: '',
+  size: '',
+  external: false,
+  download: '',
+  type: '',
+  default: 'a GCDS Link component',
+};
+
+export const External = Template.bind({});
+External.args = {
+  href: 'http://design-system.alpha.canada.ca',
+  rel: '',
+  target: '',
+  size: '',
+  external: true,
+  download: '',
+  type: '',
+  default: 'an external',
+};
+
+export const Download = Template.bind({});
+Download.args = {
+  href: 'test.jpg',
+  target: '_self',
+  size: 'regular',
+  download: 'file.pdf',
+  type: 'image/jpg',
+  default: 'a file download',
```

Comment:
I know this is just an example of a fake PDF download but to follow our own guidance can we add the file type + size in brackets to the link text, please?